tecken/tests/test_storage.py

```diff
@@ -165,39 +144,3 @@ def test_storageerror_msg():
         " operation: Forbidden"
     )
     assert str(error) == expected
-
-
-def test_StorageBucket_client():
```

Comment:
I don't understand what this is testing other than the specifics of how it's implemented. I don't think we get a lot of value out of testing implementation details like this, so I nixed it.